packages/odyssey-react-mui/src/Callout.tsx

```diff
@@ -89,7 +89,10 @@ const CallToAction = styled(MuiLink, {
   shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
```

Comment:
Also, We _may_ not need this anymore. I am not seeing these props being passed into the DOM any longer. It's possible the recent package updates fixed it 🤷

Reply:
Let's leave this for now, find out for sure if we can remove it, and then have a separate PR to remove it everywheere